doc/en/user/source/community/gdal.rst

```diff
@@ -0,0 +1,188 @@
+.. _gdal_wcs_output_format:
+
+GDAL based WCS Output Format
+============================
+
+The gdal_translate based output format leverages the availability of the gdal_translate command to allow the generation of more output formats than GeoServer can natively produce.
+The basic idea is to dump to the file system a file that gdal_translate can translate, invoke it, zip and return the output of the translation.
+
+This extension is thus the equivalent of the :ref:`OGR extension <ogr_extension>` for raster data. 
+
+
+Out of the box behaviour
+------------------------
+
+Out of the box the plugin assumes the following:
+
+* gdal_translate is available in the path
+* the GDAL_DATA variable is pointing to the GDAL data directory (which stores the spatial reference information for GDAL)
+
+In the default configuration the following formats are supported:
+
+* JPEG-2000 part 1 (ISO/IEC 15444-1)
+* Geospatial PDF
+* Arc/Info ASCII Grid
+* ASCII Gridded XYZ
+
+The list might be shorter if gdal_translate has not been built with support for the above formats (for example, the default JPEG-2000 format relies on the `JasPer-based GDAL driver <http://www.gdal.org/frmt_jpeg2000.html>`_).
+
+Once installed in GeoServer, a bunch of new supported formats will be listed in the ``ServiceMetadata`` section of the WCS 2.0 GetCapabilities document, e.g. ``image/jp2`` and ``application/pdf``.
+
+gdal_translate conversion abilities
+-----------------------------------
+
+The gdal_translate utility is usually able to convert more formats than the default setup of this output format allows for, but the exact list depends on how the utility was built from sources. To get a full list of the formats available by your ogr2ogr build just run::
+
+   gdal_translate --long-usage 
+
+and you'll get the full set of options usable by the program, along with the supported formats.
+
+For example, the above produces the following output using gdal 1.11.2 compiled with libgeotiff 1.4.0, libpng 1.6,  libjpeg-turbo 1.3.1, libjasper 1.900.1 and libecwj2 3.3::
+
+   Usage: gdal_translate [--help-general] [--long-usage]

```

Comment:
Thanks for docs!

For this you may wish to isolate the goal_translate usage into a seperate file and ".. include:" it ... it will be easier to maintain as we update gdal version numbers.